contracts/ReaperAutoCompound-Liquid-v2-Beethoven.sol

```diff
@@ -360,10 +352,10 @@ contract ReaperAutoCompound_LiquidV2_Beethoven is ReaperBaseStrategy {
             BEET_VAULT,
             type(uint256).max - IERC20(REWARD_TOKEN).allowance(address(this), BEET_VAULT)
         ); //todo is this even needed?
-        IERC20(WFTM).safeIncreaseAllowance(
-            BEET_VAULT,
-            type(uint256).max - IERC20(WFTM).allowance(address(this), BEET_VAULT)
-        ); //todo is this even needed?
+        // IERC20(WFTM).safeIncreaseAllowance(
```

Comment:
since we're giving allowance for WFTM in _addLiquidity an overflow occured without commenting these lines.

honestly I'm not sure if its worth doing individual allowances during harvest as opposed to giving all the underlying tokens max allowance in one go like we have been doing previously.

Reply:
As a safe practice, increasing allowance while harvesting is better, but for gas efficiency, I would say giving the max allowance as usual is better.
And I think we want the harvest to be as cheap as they can be